user-operator/src/main/java/io/strimzi/operator/user/Main.java

```diff
@@ -130,7 +130,7 @@ public static void main(String[] args) {
         CompositeFuture.join(clusterCaCertSecretFuture, eoKeySecretFuture)
                 .onComplete(ar -> {
                     if (ar.succeeded()) {
-                        Admin adminClient = adminClientProvider.createAdminClient(config.getKafkaBootstrapServers(),
+                        Admin adminClient = adminClientProvider.createAdminClient(new Reconciliation("user-operator-start", "Kafka", config.getNamespace(), "user-operator"), config.getKafkaBootstrapServers(),
```

Comment:
This is a bit weird ... isn't this AdminClient used by all reconciliations? Because assuming it is, this would be way too confusing if this is logged during the whole lifecycle of the operator.

Reply:
Yes, this is confusing :( The only solution I can devise is removing `reconciliation` object from `io.strimzi.operator.common.DefaultAdminClientProvider#createAdminClient` but it removes this info from some operations. Do you see any better way?

Reply:
Well, I guess you need to make it optional or something. I'm not sure how it is used in all other places.

Reply:
The _file not deleted_ ... not sure they matter that much. So maybe it can be without the reocnciliation?